static/js/studies.js

```diff
@@ -0,0 +1,140 @@
+//ajax is asyncronous so it must call the loadStudies function on success
+function getOrderedPublicationsAndLoad() {
```

Comment:
Also need to make sure the cite.html and cite.js files are deleted

Reply:
Done :)